client/app/components/api/sections/documents/DocumentView.jsx

```diff
@@ -0,0 +1,57 @@
+var React = require('react');
+var Router = require('react-router');
+var Link = Router.Link;
+var RouteHandler = Router.RouteHandler;
+var Markdown = require('react-remarkable');
+
+var fetchDocument = function (data, callback) {
+  if (data.section === "components"){
+    if (data.document === "form"){
+      callback("Hello World -FORM");
+    }
+    if (data.document === "router"){
+      callback("Hello World -router");
+    }
+  }
+}
+
+var DocumentView = React.createClass({
+  mixins: [ Router.State ],
+  getInitialState: function (){
+    return {
+      documentToRender: null
+    };
+  },
+  getDocumentsFromStore: function () {
+    var section = this.getParams().section;
+    var document = this.getParams().document;
+    fetchDocument({
+      section: section,
+      document: document
+    },function (documentToRender) {
```

Comment:
You might want to refactor this anonymous function into a method on the React.createClass input object and then pass it in that way. You'll also want to remove this listener on componentWillUnmount, which is why it would be good to refactor out into it's own method.